### DISCUSSION TITLE
Alarm activation Problem

### DISCUSSION THREAD

**Discussion by MaloryBouvier** [dgreif/ring#1261] [Q&A] [answered]
Hello everyone,

I'm having problems when activating the "Home" mode: the plug-in seems to automatically activate the "Away" mode. 
For example: I activate "Home" mode from the alarm's keypad or from HomeKit, immediately afterwards I see in the plug-in's logs: "Arming (Away) Alarm" and the alarm switches to "Away" mode.
Does anyone have the same problem as me? Where can this be coming from? I don't have any automation forcing this mode of operation and I've also tried uninstalling then reinstalling and reconfiguring the plug-in but nothing helps.

Thank you for your response

**Comment by dgreif** [comment]
@MaloryBouvier sorry for the long delay! Would you mind posting your homebridge config (minus the refresh token)?

**Reply by MaloryBouvier** [reply, marked as answer]
Hello, thank you for your reply. I don't know what happened, but after a few tries and reconfigurations the problem disappeared!
